### PR TITLE
Hand-jams Vuetify into your codebase for no reason

### PR DESCRIPTION
**Pull request by EvanSooHoo**
🔥DO NOT ACCEPT THIS PULL REQUEST🔥

This pull request will do the following to your nutrition counter:

🔥OVERCOMPLICATES deployment procedure.  Instead of viewing index.html, user must now navigate to nutrition folder, type "npm install", then type "npm run dev"
🔥DESTABILIZES code base.  Developers now depend on Vue3 and Vuetify3.  Is Vuetify3 stable?  No.  In fact, just using unsupported component v-data-table required the use of Vuetify labs, something that is NOT SUPPORTED and advised as FOR TESTING ONLY BECAUSE IT WILL BREAK 🔥🔥🔥🔥🔥🔥
🔥ADDS UNNECESSARY PACKAGES.  Do you need vue router? no.  Do you need Vuetify components?  No.  None of this is necessary and in fact it could break everything next time any of our dependencies change, which is likely to be in five minutes.  But hey, at least the color scheme is nice and the search tool is REACTIVE NOW 
🔥INCREASES cognitive complexity
🔥ELIMINATES simplicity.  How long did it take you to develop this html, css, and javascript file?  Not that long, because you didn't have to worry about tracking the latest updates on an unstable frontend component framework built on vue?  WELL NOW EVERYTHING IS MORE COMPLICATED

🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥

tl;dr please do not accept this pull request.  perserve it for history and archieve it as a reminder for how not to do web development.  May the lord of light protect us all

🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥
newone

### CODE REVIEW

**Comment by stevester94** [comment]
What the heck did you just say about Vue.js, you little HTML scrub? I'll have you know I graduated top of my class in the Vue.js Academy, and I've been involved in numerous secret Vue.js projects, and I have over 300 confirmed Vue components in my portfolio.

I am trained in the art of reactive programming, and I'm the top coder in the entire Vue.js community. You are nothing to me but just another JavaScript novice. I will wipe out your callback functions with precision the likes of which have never been seen before on this web, mark my words.

You think you can get away with saying that crap about Vue.js over the Internet? Think again, coder. As we speak, I am contacting my secret network of Vue.js developers across the world, and your IP address is being traced right now, so you better prepare for the storm, maggot. The storm that wipes out the pathetic little thing you call your frontend code. You're done, kid. I can be anywhere, anytime, and I can debug your Vue.js app in over seven hundred ways, and that's just with my bare hands.

Not only am I extensively trained in Vue.js, but I have access to the entire arsenal of Vue.js plugins and I will use them to their full extent to wipe your miserable code off the face of the Internet, you little newbie. If only you could have known what unholy reactivity your little "clever" comment was about to bring down upon you, maybe you would have held your HTML tongue.

But you couldn't, you didn't, and now you're paying the price, you Vue.js-ignorant fool. I will rain Vue directives all over you, and you will drown in the reactivity. You're DONE, kiddo.

(Ty for the pull request, but I can barely handle JS's bullshit let alone an entire frontend framework I know nothing about)
